contributors/design-proposals/resource-management/device-plugin.md

```diff
@@ -61,51 +61,72 @@ the following simple steps:
 
 # Use Cases
 
- * I want to use a particular device type (GPU, InfiniBand, FPGA, etc.)
-   in my pod.
- * I should be able to use that device without writing custom Kubernetes code.
+ * As a cluster admin I want my cluster to be aware of the devices on my nodes:
+    * My nodes should report what devices are present.
+    * My users should be able to request "device containers" (GPU, InfiniBand,
```

Comment:
will it be cleaner to simply say "devices for containers"

Reply:
For Infiniband/FPGA, do we have a roadmap on how and when these will be supported? API changes in this doc are not sufficient for these resources, in that case, should we add these in some "Futuristic" or similar section?

Reply:
> will it be cleaner to simply say "devices for containers"

"devices aware containers" should be a better term

> For Infiniband/FPGA, do we have a roadmap on how and when these will be supported? API changes in this doc are not sufficient for these resources, in that case, should we add these in some "Futuristic" or similar section?

This seems a bit out of scope of this PR, I believe an issue might be a better format than editing this PR.